app/moves/middleware.js

```diff
@@ -5,12 +5,19 @@ const {
   parseISO,
   isValid: isValidDate,
 } = require('date-fns')
-const { find, get } = require('lodash')
+const { find, get, chunk } = require('lodash')
 
 const moveService = require('../../common/services/move')
 
 const moveDateFormat = 'yyyy-MM-dd'
 
+const batchSize = 50
```

Comment:
As you mentioned in our quick call - I think it could be useful to set this in the config and have it overridable by an env var.